app/helpers/exercise_helper.rb

```diff
@@ -3,7 +3,22 @@
 module ExerciseHelper
   include LtiHelper
 
+  SETTINGS = CodeOcean::Config.new(:code_ocean)
+  EVENT_SETTINGS = SETTINGS.read[:codeocean_events] || {}
```

Comment:
NITS: Can we prefix this instance variables to make them more specific to the editor? I worried about what will happen if another helper sets an instance variable with the same name. It will be fine as long the value is the same.

Reply:
Which instance variables are you referring to? The constants? Aren't they automatically prefixed by the helper?

```rb
ExerciseHelper::SETTINGS
```